nuget/Microsoft.CodeAnalysis.FlowAnalysis.Utilities/Microsoft.CodeAnalysis.FlowAnalysis.Utilities.Package.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard1.3</TargetFramework>
     <IsPackable>true</IsPackable>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
```

Comment:
❔ This one shouldn't be suppressed right?

Reply:
no idea. What was @mavasani and your intention in this repo? None of its pack targets are standard.